content/news/release-20151224.md

```diff
@@ -0,0 +1,61 @@
+<script>
+ (function(d, t) {
+    var g = d.createElement(t),
+        s = d.getElementsByTagName(t)[0];
+    g.src = 'http://assets.gfycat.com/js/gfyajax-0.517d.js';
+    s.parentNode.insertBefore(g, s);
+}(document, 'script'));
+</script>
```

Comment:
This breaks the page layout for me (on firefox 45):  I can't select any of the text in the news post, and there is a huge empty gap at the bottom before it eventually shows the gfy overlay.